Command/DumpCommand.php

```diff
@@ -81,6 +87,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this
             ->getContainer()
             ->get('bazinga.jstranslation.translation_dumper')
-            ->dump($this->targetPath, $formats, $merge);
+            ->dump($this->targetPath, $this->getTranslationsPath($input->getOption('path')), $formats, $merge);
+    }
+
+    /**
+     * Get translations path from the router defined or use the default one.
+     */
+    private function getTranslationsPath($path)
+    {
+        $translationsPathOverriden = empty($path) ? 'translations' : $path;
+        $route = $this->getContainer()->get('router')->getRouteCollection()->get('bazinga_jstranslation_js');
```

Comment:
the dumper should require getting the pattern as argument IMO (so no dependency on the router).

And the Command class should have an option to pass this pattern. If the option is not used, it should get the pattern from the router (i.e. optional deps on the router here)